docs/develop/init.md

```diff
@@ -0,0 +1,58 @@
+# Getting Started
+
+You are now ready to learn how to create, customize and iterate on your own
+InvenioRDM instance. The previous section showed you what is the equivalent of
+a deploy compilation when working with a compiled language: it gets you the final
```

Comment:
Not your English level - it's my analogies. The way I think about the commands now, is the way I think about working on a project with a compiled language: as you develop, you recompile but with caches, no optimizations and with debug modes on. Once you are ready for a release, you compile the project without cache, with optimizations and no debug mode. You only do this step last, because it's a terrible iteration experience otherwise.

That's what my awkward words were trying to convey... Maybe the paragraph above is clearer and I should use it or maybe the analogy is not approachable enough and something else would be better?

Reply:
Sort of removed it for now.

Reply:
I'll blame it on friday afternoon hehe I didn't fully understood the analogy with the compiled language. It might be useful, but IMHO the text you put above as explanation is much more clearer :)